src/util-debug.c

```diff
@@ -1385,8 +1386,12 @@ void SCLogLoadConfig(int daemon, int verbose)
                     level_s);
                 exit(EXIT_FAILURE);
             }
+            max_level = MAX(max_level, level);
         }
 
+        /* Increase the level of extra verbosity was requested. */
+        level = MAX(min_level, level);
```

Comment:
This sets level to the max of `NOTICE` + verbosity or the configured level. IMO, the verbosity should be added to the configured level, not `NOTICE`.

Reply:
Would that still meet the requirement that -v is always info, -vv is always perf and so on?

Per the issue, we want the levels to be fixed, not relative.